usagov/usagov-2021
# Usagov 1273 accordion color

## Jira Task

https://cm-jira.usa.gov/browse/USAGOV-1273

## Description

Currently the background color of the primary accordions is #DFE1E2 and the mobile footer accordions is #EFEFEF. We are consolidating the number of grays that we use on the site. As part of this effort we are changing the primary accordions and mobile footer accordions to #F0F0F0. The hover state can remain #C9C9C9.
Primary accordions include any accordion found on content level pages with the exception of the light aqua internal/secondary accordions found on the CEO tool results page here : https://www.usa.gov/elected-officials-results?input-street=1800+F+Street+NW&input-city=Washington&input-state=DC&input-zip=20405#skip-to-h1
## Type of Changes

  - Add screenshot showing what it should look like

## Testing Instructions



### Requires New Config

### Requires New Content

### Validation Steps
- Goto http://localhost/agency-index and inspect the accordion to verify the background color is #f0f0f0
- Goto http://localhost/elected-officials-results?input-street=1800+F+Street+NW&input-city=Washington&input-state=DC&input-zip=20405#skip-to-h1 and verify the accordion background color is #f0f0f0, then click an accordion and verify the sub accordion color is #ccecf2;

## Security Review




## Reviewer Reminders
- Reviewed code changes
- Reviewed functionality
- Security review complete or not required

## Post PR Approval Instructions
Follow these steps as soon as you merge the new changes.

1. Go to the [USAGov Circle CI project](https://app.circleci.com/pipelines/github/usagov/usagov-2021).
2. Find the commit of this pull request.
3. Build and deploy the changes.
4. Update the Jira ticket by changing the ticket status to `Review in Test` and add a comment. State whether the change is already visible on [cms-dev.usa.gov](http://cms-dev.usa.gov/) and [beta-dev.usa.gov](http://beta-dev.usa.gov/), or if the deployment is still in process.
